core/src/main/java/org/apache/iceberg/rest/ErrorHandlers.java

```diff
@@ -100,15 +123,30 @@ private static Consumer<ErrorResponse> baseNamespaceErrorHandler() {
         case 422:
           throw new RESTException("Unable to process: %s", error.message());
       }
-    };
+
+      super.accept(error);
+    }
   }
 
   /**
    * Request error handler that handles the common cases that are included with all responses, such
    * as 400, 500, etc.
    */
-  public static Consumer<ErrorResponse> defaultErrorHandler() {
-    return error -> {
+  private static class DefaultErrorHandler extends ErrorHandler {
```

Comment:
I think we have to leave this public since it was public before.  Maybe we can added a @Deprecated with a comment to move to private after the 1.1 release?.

Reply:
It is still public, the diff is a little wonky